lusSTR/wrappers/filter.py

```diff
@@ -66,7 +90,7 @@ def get_filter_metadata_file():
     filter_marker_data = json.load(fh)
 
 
-def process_strs(dict_loc, datatype, seq_col, brack_col):
+def process_strs(dict_loc, datatype, seq_col, brack_col, sex):
```

Comment:
It doesn't look like the `sex` argument is used in this function. Did I miss something, or is it really unnecessary?

Reply:
Yup, you're right! I've removed it.